Source/Krypton Components/Krypton.Toolkit/Controls Visuals/Notification Toast/User Input/Left to Right/Normal/VisualToastNotificationComboBoxUserInputForm.cs

```diff
@@ -94,9 +94,9 @@ private void UpdateIcon()
             }
         }
 
-        public new DialogResult ShowDialog(IWin32Window owner)
+        public new DialogResult ShowDialog()
```

Comment:
There needs to be a version that also takes the owner in order for windows to properly tab through the windows.

Reply:
Going to do this in a separate PR this week (going to 'borrow' code from the existing input dialog) :)